hazelcast/pom.xml.orig

```diff
@@ -0,0 +1,779 @@
+<!--
```

Comment:
Is this file some merge conflict leftover?

Reply:
Good catch. will fix it